releasenotes/releasenotes_develop.md

```diff
@@ -58,11 +70,18 @@ Implemented **Keptn spec** version: [0.2.1](https://github.com/keptn/spec/tree/0
 <details><summary>Bridge</summary>
 <p>
 
+- OAuth/OpenID connect based login for Keptn bridge [3448](https://github.com/keptn/keptn/issues/3448)
```

Comment:
Please add `OAuth` to the `expect.txt` file -- something's wrong w/ the bot -- I need to figure out why it flaked out.

Reply:
Oh, you did, sigh. Sorry.

Reply:
Fwiw, the buggy error reporting from the spell checker is already fixed in prerelease (yay). I need to work on writing release notes and releasing -- just like you folks :-)